### DISCUSSION TITLE
Emails not sent from Toshiba and Kyocera printers.

### DISCUSSION THREAD

**Discussion by samykalem** [postalserver/postal#2733] [Help with using Postal]
## Describe the bug
Toshiba printers and Kyocera printers cant send emails using postal service. Both printers believe email was sent but in logs i see that they never send the final commands. On another smtp service. When they hang the system automatically tries to send the email as is.

Proper smtp information is used. 

## To Reproduce
Set correct credentials 
Scan to email

## Expected behaviour
Email being sent.

## LOGs:
smtp_1      | [9] [2023-12-28T13:59:41.648] DEBUG -- : [XYK4AU]    Connection opened from ::ffff:XXX.XXX.XXX.XXX
smtp_1      | [9] [2023-12-28T13:59:41.648] DEBUG -- : [XYK4AU]    Client identified as ::ffff:XXX.XXX.XXX.XXX
smtp_1      | [9] [2023-12-28T13:59:41.655] DEBUG -- : [XYK4AU] <= EHLO Office500.XXX.local
smtp_1      | [9] [2023-12-28T13:59:41.676] DEBUG -- : [XYK4AU] => 250-My capabilities are
smtp_1      | [9] [2023-12-28T13:59:41.676] DEBUG -- : [XYK4AU] => 250-STARTTLS
smtp_1      | [9] [2023-12-28T13:59:41.676] DEBUG -- : [XYK4AU] => 250 AUTH CRAM-MD5 PLAIN LOGIN
smtp_1      | [9] [2023-12-28T13:59:41.800] DEBUG -- : [XYK4AU] <= STARTTLS
smtp_1      | [9] [2023-12-28T13:59:41.800] DEBUG -- : [XYK4AU] => 220 Ready to start TLS
smtp_1      | [9] [2023-12-28T13:59:41.875] DEBUG -- : [XYK4AU] <= EHLO Office500.XXX.local
smtp_1      | [9] [2023-12-28T13:59:41.895] DEBUG -- : [XYK4AU] => 250-My capabilities are
smtp_1      | [9] [2023-12-28T13:59:41.895] DEBUG -- : [XYK4AU] => 250 AUTH CRAM-MD5 PLAIN LOGIN
smtp_1      | [9] [2023-12-28T13:59:41.966] DEBUG -- : [XYK4AU] <= AUTH PLAIN [redacted]
smtp_1      | [9] [2023-12-28T13:59:42.457] DEBUG -- : [XYK4AU] => 235 Granted for XXXX/XXX
smtp_1      | [9] [2023-12-28T13:59:42.461] DEBUG -- : [XYK4AU] <= MAIL From:<scanner@XXXXXXXXXXXX.com>
smtp_1      | [9] [2023-12-28T13:59:42.461] DEBUG -- : [XYK4AU] => 250 OK
smtp_1      | [9] [2023-12-28T13:59:42.466] DEBUG -- : [XYK4AU] <= RCPT To:<REDACTED@Xxxxxxxxxxx.com>
smtp_1      | [9] [2023-12-28T13:59:42.466] DEBUG -- : [XYK4AU] Added external address 'Redacted@XXXXXXXXXX.com'
smtp_1      | [9] [2023-12-28T13:59:42.466] DEBUG -- : [XYK4AU] => 250 OK
smtp_1      | [9] [2023-12-28T13:59:42.470] DEBUG -- : [XYK4AU] <= DATA
smtp_1      | [9] [2023-12-28T13:59:42.470] DEBUG -- : [XYK4AU] => 354 Go ahead
smtp_1      | [9] [2023-12-28T14:15:54.845] DEBUG -- : [XYK4AU]    Connection closed

**Comment by willpower232** [comment]
```
smtp_1 | [9] [2023-12-28T13:59:42.470] DEBUG -- : [XYK4AU] => 354 Go ahead
smtp_1 | [9] [2023-12-28T14:15:54.845] DEBUG -- : [XYK4AU] Connection closed
```

nearly 16 minutes to upload the message content? That seems like a long time. Is it a highly detailed scan or something? Can you see this message in the Postal UI?
